docs/decisions/0018-standarize-django-po-files.rst

```diff
@@ -0,0 +1,193 @@
+Standardize ``django.po`` and ``djangojs.po`` files
+===================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+#. The edx-platform splits its translations files to ``mako-studio.po``,
+   ``django-partial.po`` among 11 files in total to make it possible for
+   translators to focus on learner-facing text
+   and de-prioritize educator-facing text. The idea was that LMS would be
+   translated into more languages than CMS (Studio) would.
+
+#. The edx-platform uses the ``i18n_tools segment`` to split the ``django.po``
+   and ``djangojs.po`` files into smaller files. This is done with the goal to
+   make it easier for translators to translate the files.
+
+#. When pulling the files from Transifex, the files are bundled again into
+   ``django.po`` and ``djangojs.po`` files.
+
+#. When cutting a new Open edX named-release, the files are bundled as well
+   into ``django.po`` and ``djangojs.po`` files.
+
+#. The `FC-0012 project <https://openedx.atlassian.net/l/cp/XGS0iCcQ>`_
+   which implement `Translation Infrastructure update OEP-58 <https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification>`_
+   is in progress which means that all translation files are going to live in
+   the new
+   `openedx-translations Transifex project <https://app.transifex.com/open-edx/openedx-translations/>`_.
+
+#. The `openedx-translation GitHub repository <https://github.com/openedx/openedx-translations>`_
+   is going to sync the translation between repos and Transifex as well as
+   store the translations files in centralized location as accepted in
+   `Translation Infrastructure update OEP-58 <https://open-edx-proposals.readthedocs.io/en/latest/architectural-decisions/oep-0058-arch-translations-management.html#specification>`_.
+
+#. Several special steps are needed to for the ``extract-translation-source-files.yml``
+   to extract the files from the edx-platform repo. This seems redundant and and
+   it should be possible to simplify the process.
+
+Decision: Combine in two files (``django.po`` and ``djangojs.po``)
+------------------------------------------------------------------
+
+This implementation is one of many "special" ways that the platform handles
+i18n among other fields that makes it hard to understand the platform for
+new-comers.
+
+In this decision we argue that the cost outweighs the benefits and propose
+to reduce the number of Transifex resources in the
+``openedx-translations`` Transifex project, which simplifies the work of both
+Build Test Release and Transifex working groups.
```

Comment:
this feels more like consequence than decision. It's also not explained how or why this will simplify BTR and translations WG work.